Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/SingleZone/VAV/Controller.mo

```diff
@@ -281,6 +281,12 @@ block Controller
     final unit="1") = 0
     "Physically fixed minimum position of the return air damper"
     annotation(Dialog(tab="Economizer", group="Commissioning"));
+  parameter Integer cooDemLimLevCon=Buildings.Controls.OBC.ASHRAE.G36_PR1.Types.DemandLimitLevels.cooling0
+    "Cooling demand limit level"
+    annotation (Dialog(tab="Advanced"));
+  parameter Integer heaDemLimLevCon=Buildings.Controls.OBC.ASHRAE.G36_PR1.Types.DemandLimitLevels.heating0
+    "Heating demand limit level"
+    annotation (Dialog(tab="Advanced"));
 
   Buildings.Controls.OBC.CDL.Interfaces.RealInput TOut(
```

Comment:
Please expose the warUp and cooDow parameters as inputs of the block and update the SZVAV G36 example model.

Reply:
Propagated.